example/example-node.ts

```diff
@@ -2,7 +2,7 @@ import { TezosToolkit } from '../packages/taquito/src/taquito';
 import { RpcClient } from '../packages/taquito-rpc/src/taquito-rpc';
 import { castToString } from '../packages/taquito-rpc/src/utils/utils';
 
-const provider = 'https://api.tez.ie/rpc/florencenet';
+const provider = 'https://api.tez.ie/rpc/granadanet';
```

Comment:
The example won't work because KT1W1jh5C5NbcVVvpnBLQT9ekMbR5a8fg6mc is on florencenet

Reply:
i will fx example cases in next pr

Reply:
for now, I will put them back to florencenet so they still work